src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/InMemoryTokenCacheAccessor.cs

```diff
@@ -16,23 +16,25 @@ namespace Microsoft.Identity.Client.PlatformsCommon.Shared
     /// Keeps the token cache dictionaries in memory. Token Cache extensions
     /// are responsible for persistance.
     /// </summary>
-    /// <remarks>See this post for efficient use of Concurrent Dictionary http://geekswithblogs.net/simonc/archive/2012/02/22/inside-the-concurrent-collections-concurrentdictionary.aspx</remarks>
     internal class InMemoryTokenCacheAccessor : ITokenCacheAccessor
     {
+        // perf improvement: intialize the capacity and concurrency level 
+        // since most websites would use distributed caching, these dictionaries would mostly hold a single item
         private readonly ConcurrentDictionary<string, MsalAccessTokenCacheItem> _accessTokenCacheDictionary =
-            new ConcurrentDictionary<string, MsalAccessTokenCacheItem>();
+            new ConcurrentDictionary<string, MsalAccessTokenCacheItem>(1, 1);
```

Comment:
the numbers here are: 

- concurrency level, which is mostly 1 since token caches are not normally shared
- capacity, which is also mostly 1 in confidential client apps, where perf is critical